cmd/mattermost/commands/plugin.go

```diff
@@ -184,3 +215,83 @@ func pluginListCmdF(command *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func pluginPublicKeysCmdF(command *cobra.Command, args []string) error {
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer a.Shutdown()
+
+	verbose, err := command.Flags().GetString("verbose")
+	if err != nil {
+		return errors.New("failed reading verbose. Error: " + err.Error())
+	}
+
+	pluginPublicKeysResp, appErr := a.GetPluginPublicKeys()
+	if appErr != nil {
+		return errors.New("Unable to list public keys. Error: " + appErr.Error())
+	}
+
+	if verbose == "" {
+		for _, publicKey := range pluginPublicKeysResp {
```

Comment:
Nit: Can we move the `verbose` check within the `for` loop to simplify this block and not repeat the for loop definition?

Reply:
I'd rather leave as is. For me this is more readable and there is no check in every iteration of the loop.